geetools/ImageCollection.py

```diff
@@ -2,16 +2,25 @@
 from __future__ import annotations
 
 import uuid
+from datetime import datetime as dt
 
 import ee
 import ee_extra
 import requests
 import xarray
 from ee import apifunction
+from matplotlib.axes import Axes
 from xarray import Dataset
 from xee.ext import REQUEST_BYTE_LIMIT
 
-from geetools.accessors import register_class_accessor
+from .accessors import register_class_accessor
+from .utils import plot_data
+
+PY_DATE_FORMAT = "%Y-%m-%dT%H-%M-%S"
+"The python format to use to parse dates coming from GEE."
+
+JS_DATE_FORMAT = "YYYY-MM-dd'T'HH-mm-ss"
+"The javascript format to use to burn date object in GEE."
```

Comment:
it's manually build to meet requirements from band naming (reason why there are no ":" for hours as per ISO format). I named it JS as per "use it with the javascript methods" I could rename it "EE_DATE_FORMAT" if you want to avoid confusion.

Reply:
I don't think there will be any confusion since this is only reached by devs and should not reach end users. It was just an irrelevant comment. There could also be a link to https://joda-time.sourceforge.net/apidocs/org/joda/time/format/DateTimeFormat.html, which is the formatting guide.